connect/startup.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 set -e
-set -x
+if [[ "${STARTUP_DEBUG_MODE}" -eq 1 ]]; then
```

Comment:
Explicitly default this to OFF to prevent comparison problems.
```suggestion
if [[ "${STARTUP_DEBUG_MODE:-0}" -eq 1 ]]; then
```